src/pages/login/index.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import {useState} from 'react';
+import ApiButton from '../../components/ApiButton';
+import styled from 'styled-components';
+import Layout from '../../components/Layout';
+import axios from 'axios';
+import {API} from '../../constants/api';
+import {useRouter} from 'next/router';
+import {TOKEN} from '../../constants/token';
+
+const Container = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 240px;
+`;
+
+const Nickname = styled.div`
```

Comment:
전 div에는 Container를 주로 붙이는 편인데 한번 같이 이야기 해보죠
```suggestion
const NicknameContainer = styled.div`
```

Reply:
네 좋습니다😬

Reply:
NicknameContainer를 활용하기로 했습니다.